sparkmagic/sparkmagic/magics/remotesparkmagics.py

```diff
@@ -112,7 +114,7 @@ def spark(self, line, cell="", local_ns=None):
         # info
```

Comment:
We need to update the documentation in the comment above to add the `-t` parameter for the `add` subcommand.